bool stop_robot			# Stop the robot
---
bool stopped_robot		# Has the robot stopped?